app/src/main/java/com/kickstarter/ui/activities/compose/projectpage/AddOnsContainer.kt

```diff
@@ -48,14 +42,13 @@ private fun AddOnsContainerPreview() {
         AddOnsContainer(
             title = "This Is A Test",
             amount = "$500",
-            shippingAmount = "$5 shipping each",
```

Comment:
Do not remove the Shipping amount, there will be project not opted on the feature flag, that might potentially still charge shipping.